src/type_filter.rs

```diff
@@ -128,24 +130,28 @@ where
         self.is_null
     }
 
+    fn is_not_null(&self) -> Option<bool> {
+        self.is_not_null
+    }
+
     fn contains(&self) -> Option<String> {
-        panic!("FilterType does not support contains")
+        None
     }
 
     fn starts_with(&self) -> Option<String> {
-        panic!("FilterType does not support starts_with")
+        None
     }
 
     fn ends_with(&self) -> Option<String> {
-        panic!("FilterType does not support ends_with")
+        None
     }
 
     fn like(&self) -> Option<String> {
-        panic!("FilterType does not support like")
+        None
     }
 
     fn not_like(&self) -> Option<String> {
-        panic!("FilterType does not support not_like")
+        None
```

Comment:
Design choice: no-operation or a explicit panic.

I second the no-operation as changed.

Reply:
Okay, so if I understand you correctly that means that you prefer no-operation and I should keep these changes? :smile: